bigflow/build.py

```diff
@@ -102,7 +103,7 @@ class BuildCommand(distutils.cmd.Command):
         ]
 
         def initialize_options(self) -> None:
-            self.start_time = now()
+            self.start_time = datetime.now().strftime("%Y-%m-%d %H:00:00")
```

Comment:
I prefer the previous solution

Reply:
IMO explicit is much better. It is extremely unclear what is going here:
 - what type does 'now' return?
 - does it include date, time, or "hours without minutes and seconds"
As an option we can refactor datetime format into some named const, but I still prefer explicit format in this case.